README.md

```diff
@@ -1,111 +1,213 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
+<div align="center">
+
+  <img src="img/letter.jpg" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>To-Do List</b></h3>
+
+</div>
 
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 AwesomeBooksES6 <a name="about-project"></a>
+# 📖 To-Do List <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
 
-**AwesomeBooksES6** is a simple website that allows users to add/remove books from a list. This project is part of the Microverse curriculum.
+**To-Do List** is a simple TODO list application created using webpack
 
 ## 🛠 Built With <a name="built-with"></a>
 
-- HTML
-- CSS
-- JavaScript
+### Tech Stack <a name="tech-stack"></a>
+
+<ul>
+    <li><a href="">HTML</a></li>
+    <li><a href="">CSS</a></li>
+    <li><a href="">JavaScript</a></li>
+  </ul>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Using Webpack for project developement**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](example.com)
+> Add a link to your deployed project.
+
```

Comment:
- Dear @Habtamu210, this lines are a suggestion/direction and should not be part of your readme. Please concider removing/replacing all.